template-parts/content/content-not-found.php

```diff
@@ -9,12 +9,12 @@
  */
 
 ?>
-<article class="entry">
-	<header class="entry__header">
-		<h1 class="entry__title"><?php esc_html_e( 'Not Found', 'cata' ); ?></h1>
+
+<article <?php post_class( 'wp-site-blocks wp-block-post-content' ); ?>>
```

Comment:
Does `post_class` do anything on a "not found" page?

Reply:
It outputs the classes like it does on other pages, but I can change it to `class="wp-site-blocks"` instead if you'd prefer

Reply:
OK it can stay